### PR TITLE
Trace branding

### PR DESCRIPTION
**Pull request by Voronenko**
### What does this PR do?

Introduces hostname, pid, starttime spans,
adjusts configuration branding,
returns passing molecule tests

### CODE REVIEW

**Comment by TizianoPerrucci** [comment]
@Voronenko so downgrading ansible and increasing `max_test_time` solves the issue ?

Can you extract that in its own PR please, because:
- it has nothing to do with trace agent
- I want to run pipeline few times

**Comment by Voronenko** [comment]
Extracted to  https://github.com/StackVista/stackstate-agent/pull/91

File: cmd/agent/install_script.sh [hunk 1]
```diff
@@ -203,6 +203,8 @@ if [ $sts_url ]; then
     sts_url_esc=$(sed 's/[/.&]/\\&/g' <<<"$sts_url")
     print_blu "* Adding StackState url to the Agent configuration: $CONF\n"
     $sudo_cmd sh -c "sed -i 's/sts_url:.*/sts_url: $sts_url_esc/' $CONF"
+    $sudo_cmd sh -c "sed -i 's/process_sts_url:.*/process_sts_url: $sts_url_esc/' $CONF"
```
Comment (TizianoPerrucci) [inline cmd/agent/install_script.sh]: this should not be needed, covered by previous replace `sts_url`

File: cmd/agent/install_script.sh [hunk 2]
```diff
@@ -203,6 +203,8 @@ if [ $sts_url ]; then
...
+    $sudo_cmd sh -c "sed -i 's/apm_sts_url:.*/apm_sts_url: $sts_url_esc/' $CONF"
```
Comment (TizianoPerrucci) [inline cmd/agent/install_script.sh]: this should not be needed, covered by previous replace `sts_url`

File: pkg/config/config_template.yaml [hunk 3]
```diff
@@ -831,17 +831,19 @@ network_tracer_config:
 {{- if .TraceAgent }}
 # Trace Agent Specific Settings
 #
-# apm_config:
+apm_config:
 #   Whether or not the APM Agent should run
-#   enabled: true
+  enabled: true
+  apm_sts_url:
+  apm_dd_url:
```
Comment (TizianoPerrucci) [inline pkg/config/config_template.yaml]: we should only keep `apm_sts_url`

**Pull request merged**